js/package.json

```diff
@@ -8,7 +8,7 @@
     "@jupyter-widgets/base": "^1.2.2",
     "@jupyter-widgets/controls": "^1.4.2",
     "@jupyter-widgets/html-manager": "^0.15.2",
-    "@jupyter-widgets/jupyterlab-manager": "^0.39.1",
+    "@jupyter-widgets/jupyterlab-manager": "^0.40.0",
```

Comment:
you probably need to update html-manager to 0.16 then?

Reply:
Good question. I thought html-manager was not a dependency anymore, but it is, only for the requireLoader
https://github.com/QuantStack/voila/blob/ab241427955d6dd917e74962d6365937f9c3fffe/js/src/manager.js#L10
So it's fine I think. I'd rather see that requireLoader go into main.js, and it being an argument of the widget manager, so also this part is pluggable.

Reply:
Yeah, sorry for a minute I thought that html-manager depended on jupyterlab-manager.